crates/net/network/Cargo.toml

```diff
@@ -69,9 +69,12 @@ reth-interfaces = { path = "../../interfaces", features = ["test-utils"] }
 reth-provider = { path = "../../storage/provider", features = ["test-utils"] }
 reth-tracing = { path = "../../tracing" }
 reth-transaction-pool = { path = "../../transaction-pool", features = ["test-utils"] }
+reth-net-test-utils = { path = "../test-utils" }
```

Comment:
Do we need this to be a separate crate?

Reply:
we don't need this to be a separate crate, we can make it a submodule of `network`